crates/cli/src/commands.rs

```diff
@@ -228,6 +237,17 @@ option_group! {
     pub enum JsOption {
         /// Whether to redirect the output of console.log to standard error.
         RedirectStdoutToStderr(bool),
+        /// Whether to enable the `Javy.JSON` builtins.
+        JavyJson(bool),
+        /// Whether to enable the `Javy.readSync` and `Javy.writeSync` builtins.
+        JavyStreamIo(bool),
+        /// Whether to override the `JSON.parse` and `JSON.stringify`
+        /// implementations with an alternative, more performant, SIMD based
+        /// implemetation.
+        OverrideJsonParseAndStringify(bool),
```

Comment:
Should this rather be called `SimdJsonBuiltins` or something along those lines?

Reply:
Great suggestion, done!